src/frontends/onnx/tests/runtime/ie/unit_test.manifest

```diff
@@ -412,3 +412,6 @@ IE_CPU.onnx_model_top_k_repeating_unsorted
 
 # Accuracy regression - Ticket 105909
 IE_CPU.onnx_model_attention_qkv_hidden_sizes
+
+# Ticket: 111110, ScatterElementsUpdate-12 (not fully implemented)
+IE_CPU.onnx_model_scatter_elements_none_neg_ind_opset18
```

Comment:
do we have downgrade transformation for plugins that do not support new version?

Reply:
Of course, it is already on master: https://github.com/openvinotoolkit/openvino/blob/feab0ab75e9c4a567091461d70de4cf3eb4686c3/src/common/transformations/src/transformations/op_conversions/convert_scatter_elements_update12_downgrade.cpp#L13

Reply:
> do we have downgrade transformation for plugins that do not support new version?

yes